scripts/generateForDay.sh

```diff
@@ -180,11 +180,10 @@ if [ "${DO_KEOGRAM}" = "true" ] ; then
 	if [ "${TYPE}" = "GENERATE" ]; then
 		CMD="'${ALLSKY_HOME}/keogram' ${SIZE_FILTER} -d '${DATE_DIR}' -e ${EXTENSION} -o '${UPLOAD_FILE}' ${KEOGRAM_EXTRA_PARAMETERS}"
 		generate "Keogram" "keogram" "${CMD}"
-		let EXIT_CODE=${EXIT_CODE}+${?}
 	else
 		upload "Keogram" "${UPLOAD_FILE}" "${KEOGRAM_DIR}" "${KEOGRAM_FILE}" "${KEOGRAM_DESTINATION_NAME}" "${WEB_KEOGRAM_DIR}"
-		let EXIT_CODE=${EXIT_CODE}+${?}
 	fi
+	[ $? -ne 0 ] && let EXIT_CODE=${EXIT_CODE}+1
```

Comment:
The behavior has now changed from a sum to a counter. If you use +1, +2, +4,... it would be a bit coded status

Reply:
The failure exit codes of all (?) the scripts don't really matter since the invokers only check for 0 and not-0.  I suppose I could have used `[ $? -ne 0 ] && let EXIT_CODE=1`, but the current way allows the possibility for generateForDay.sh to check for how many commands failed.